sass/components/_all.scss

```diff
@@ -0,0 +1 @@
+@forward 'search-box';
```

Comment:
sao phải dùng forward ở đây nhỉ, file _all này mục đích để làm gì e ?

Reply:
file đó em để load toàn bộ component trong folder để chỉ cần load file all thôi ạ.